### PR TITLE
Copy file path

### PR DESCRIPTION
**Pull request by yanganto**
It changes the following:
- allow copy file path in revision files
- allow copy file path in status tree

I followed the checklist:
- [X] This feature is not a git operation, I think we do not need a test on copy feature.
- [X] I ran `make check` without errors
- [X] I tested the overall application
- [X] I added an appropriate item to the changelog

### CODE REVIEW

File: src/components/status_tree.rs [hunk 1]
```diff
@@ -302,6 +302,21 @@ impl StatusTreeComponent {
 			selection_offset_visible,
 		)
 	}
+
+	// Copy the real path of selected file to clickboard
+	fn copy_file_path(&self) {
+		if let Some(item) = self.selection() {
+			if crate::clipboard::copy_string(&item.info.full_path)
```
Comment (extrawurst) [inline src/components/status_tree.rs:309]: why not use `try_or_popup` here aswell?

File: src/components/status_tree.rs [hunk 2]
```diff
@@ -481,6 +496,9 @@ impl Component for StatusTreeComponent {
 						}
 					}
 					Ok(EventState::Consumed)
+				} else if key_match(e, self.key_config.keys.copy) {
```
Comment (extrawurst) [inline src/components/status_tree.rs:499]: if we add a new event handler we should generally also add a new entry to the command bar

File: src/components/revision_files.rs [hunk 3]
```diff
@@ -515,6 +516,15 @@ impl Component for RevisionFilesComponent {
 					);
 					return Ok(EventState::Consumed);
 				}
+			} else if key_match(key, self.key_config.keys.copy) {
```
Comment (extrawurst) [inline src/components/revision_files.rs:519]: if we add a new event handler we should generally also add a new entry to the command bar

**Review by extrawurst — CHANGES_REQUESTED** [review]

File: src/components/status_tree.rs (hunk 1 above)
Comment (yanganto) [reply src/components/status_tree.rs:309]: The data struct of this component is different from others, so we can not use this macro here.

**Pull request merged**